Formula/clojure-lsp.rb

```diff
@@ -29,16 +29,15 @@ def install
   end
 
   test do
-    require "open3"
+    input =
+      "Content-Length: 152\r\n" \
+      "\r\n" \
+      "{\"jsonrpc\":\"2.0\",\"id\":0,\"method\":\"initialize\",\"params\":{\"" \
+      "processId\":88075,\"rootUri\":null,\"capabilities\":{},\"trace\":\"ver" \
+      "bose\",\"workspaceFolders\":null}}\r\n"
```

Comment:
Please use a heredoc to avoid the overuse of escape characters.

Reply:
Sure, will change it when the tests pass. So far, we discovered that the process is not returning 0, but 126 ("If a command is found but is not executable, the return status is 126").